app/controllers/CandidateAllocationController.scala

```diff
@@ -78,9 +78,14 @@ trait CandidateAllocationController extends BaseController {
     }
   }
 
-  def findCandidatesEligibleForEventAllocation(assessmentCenterLocation: String): Action[AnyContent] = Action.async {
+  def findCandidatesEligibleForEventAllocation(
+    assessmentCenterLocation: String,
```

Comment:
You might hate me for this, can this be `assessmentCentreLocation`? So we can consistently spell Centre the same throughout?